src/api/todos.js

```diff
@@ -0,0 +1,14 @@
+const BASE_URL = 'https://mate-api.herokuapp.com';
+
```

Comment:
this file better is called `api.js`. Here you have logic not only for fetching todos but also for fetching users. So the name `todos.js` does not actually mean what it should do.